src/steth/scWETHv2.sol

```diff
@@ -60,7 +60,9 @@ contract scWETHv2 is BaseV2Vault {
 
     constructor(address _admin, address _keeper, WETH _weth, Swapper _swapper, PriceConverter _priceConverter)
         BaseV2Vault(_admin, _keeper, _weth, _priceConverter, _swapper, "Sandclock WETH Vault v2", "scWETHv2")
-    {}
+    {
+        zeroExSwapWhitelist[ERC20(C.WSTETH)] = true;
```

Comment:
what about weth?

Reply:
Its added in the constructor of the BaseV2Vault contract.